config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  get 'greetings/random_greeting', to: 'greetings#random_greeting'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "Home#index"
```

Comment:
- You have done well so far 👍, but the previous reviewer suggested that you should have a root route with a corresponding controller with one action. This root route will serve as an entry point for your application. Please fix this 🙏